Makefile

```
@@ -98,15 +98,10 @@ git-commit: ## commit outstading git changes and push to remote
 		git commit -m "[ci skip] Automated publish for $(GITHUB_SHA)" || exit 0
 	@cd $(LOCAL_PATH) && git push -u publisher master
 
-hook/%: export COMMIT_MSG?=$(shell git log -1 --pretty=%B)
-hook/%: export GITHUB_SHA?=$(shell git rev-parse HEAD)
-hook/%: export WIKI_PATH?=../wiki
+hook/%: WIKI_PATH?=../wiki
 hook/%: ## run post-build hooks for an image
-	BUILD_TIMESTAMP="$$(date -u +%FT%TZ)" \
-	DOCKER_REPO="$(OWNER)/$(notdir $@)" \
-	IMAGE_NAME="$(OWNER)/$(notdir $@):latest" \
-	IMAGE_SHORT_NAME="$(notdir $@)" \
-	$(SHELL) $(notdir $@)/hooks/run_hook
+	./tagging/tag_image.py --short-image-name "$(notdir $@)" --owner "$(OWNER)" && \
+	./tagging/create_manifests.py --short-image-name "$(notdir $@)" --owner "$(OWNER)" --wiki-path "$(WIKI_PATH)"
```

It would be cleaner to use this tool as a module. To turn it into a module just need to add an `__init__.py` and to turn import into relative imports for example `from docker_runner` -> `from .docker_runner`.

```suggestion
	python -m tagging.tag_image --short-image-name "$(notdir $@)" --owner "$(OWNER)" && \
	python -m tagging.create_manifests --short-image-name "$(notdir $@)" --owner "$(OWNER)" --wiki-path "$(WIKI_PATH)""$(WIKI_PATH)"
```

---

I applied your suggestion.
The only difference is that I wrote `python3`. It won't work on my mac with simple python.

---

@romainx now I can't run my script as usual.
Is there a way to have both things?

```
~/D/o/d/tagging > ./tag_image.py --short-image-name datascience-notebook --owner jupyter
Traceback (most recent call last):
  File "/Users/asalikhov/Documents/oss_repos/docker-stacks/tagging/./tag_image.py", line 7, in <module>
    from .docker_runner import DockerRunner
ImportError: attempted relative import with no known parent package
```

---

@mathbunnyru unfortunately I don't think it is possible to mix module and script at least with a relative import.

---

Ok, that's a bit unfortunate.
Could you please explain me why the module approach is better here?
Sorry, I don't know well the python packaging model.